metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -509,10 +531,25 @@ def _process_database(
 
             yield from self._process_schema(conn, snowflake_schema, db_name)
 
+        if self.config.extract_tags is not TagOption.skip:
+            snowflake_db.tags = self.get_tags_on_object(
+                conn=conn, db_name=db_name, domain="database"
+            )
+
+            if self.config.include_technical_schema:
+                if snowflake_db.tags:
+                    for tag in snowflake_db.tags:
+                        yield from self._process_tag(tag)
+
     def _process_schema(
         self, conn: SnowflakeConnection, snowflake_schema: SnowflakeSchema, db_name: str
     ) -> Iterable[MetadataWorkUnit]:
         schema_name = snowflake_schema.name
+
```

Comment:
Again, why is this same code repeated below ? The lower one makes more sense as it considers extract_tags config.